pkg/templates/templates.go

```diff
@@ -50,3 +56,245 @@ func CreatePullRequestFromTemplate(params CreatePullRequestFromTemplateParams, r
 
 	return nil
 }
+
```

Comment:
All of this is moved code. The only change is adding the `kind` type to template listing.